DecimalFormatTest.testFormat is locale-dependent

Stack trace when run with `LANG=es_ES.UTF-8`:

```
Exception in thread "main" java.lang.AssertionError: Expected 4.3 unit, but was: 4,3 unit
        at malva.TestCase.fail(TestCase.java:95)
        at malva.TestCase.assertEquals(TestCase.java:81)
        at malva.java.text.DecimalFormatTest.testFormat(DecimalFormatTest.java:11)
        at malva.java.text.DecimalFormatTest.main(DecimalFormatTest.java:16)
malva/java/text/DecimalFormatTest: Test FAILED
```

This can probably be fixed by explicitly using a `DecimalFormatSymbols` instance for a fixed locale.

--- a/src/malva/java/text/DecimalFormatTest.java
+++ b/src/malva/java/text/DecimalFormatTest.java
@@ -1,15 +1,22 @@
 package malva.java.text;
 
 import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.util.Locale;
 
 import malva.TestCase;
 
 public class DecimalFormatTest extends TestCase {
   public static void testFormat() {
     // Format with suffix
-    DecimalFormat df = new DecimalFormat("0.0 unit");
+    DecimalFormat df = new DecimalFormat("0.0 unit", new DecimalFormatSymbols(Locale.US));
     assertEquals("4.3 unit", df.format(4.3));
     assertEquals("-4.3 unit", df.format(-4.3));
+
+    DecimalFormatSymbols spanishSymbols = new DecimalFormatSymbols(new Locale("es", "ES"));
+    df.setDecimalFormatSymbols(spanishSymbols);
+    assertEquals("4,3 unit", df.format(4.3));
+    assertEquals("-4,3 unit", df.format(-4.3));
   }
 
   public static void main(String[] args) {

